_posts/2015-08-11-tsuru-images-with-packer.markdown

```diff
@@ -0,0 +1,45 @@
+---
+layout: post
+status: publish
+published: false
+title: tsuru easier to test (spoiler: packer is awesome).
+author:
+  display_name: Tarsis Azevedo
+  url: https://github.com/tarsisazevedo
+date: '2015-08-11 14:30:00 -0300'
+date_gmt: '2015-08-11 14:30:00'
+categories:
+- Images
+tags:
+- images packer
+comments: []
+---
+
+One of our goals is make tsuru easy to people who wants to try it for
+the first time. To do it we are creating all-in-one aws and virtualbox vagrant
+box images of tsuru with [packer](https://packer.io). These images are based on
+stable and nightly packages.
+
+## packer + tsuru now
+
+Tsuru all-in-one images are builded as amazon ami and as vagrant virtualbox
+box. These images are generated with [tsuru-now script](https://github.com/tsuru/now) configured to have the python platform installed by
+default and pre-receive deploy method using archive server.
```

Comment:
s/builded/built/
s/amazon ami/Amazon AMI/
s/vagrant virtualbox/Vagrant VirtualBox/

The second sentence could be:

	We're using [tsuru-now](https://github.com/tsuru/now) to build the
	images. They come bundled with the Python platform, and with tsuru
	configured to use the pre-receive hook on top of
	[archive-server](https://github.com/tsuru/archive-server).